tracing-log/src/lib.rs

```diff
@@ -222,9 +236,7 @@ impl AsTrace for log::Level {
 /// A simple "logger" that converts all log records into `tracing` `Event`s,
 /// with an optional level filter.
 #[derive(Debug)]
-pub struct LogTracer {
-    filter: log::LevelFilter,
-}
+pub struct LogTracer;
```

Comment:
Let's add a private field to this so that it can't be publicly constructed. This would allow us to add new fields to this type in the future without causing a breaking change.

For example:
```rust
pub struct LogTracer {
    _p: (),
}
```

Otherwise, `LogTracer`s could be constructed by users, and adding a new field to it in the future would break this.

Reply:
If we do this, we should add `LogTracer::new` and possibly `LogTracer::default` so that users can construct new `LogTracer`s.